web/satellite/src/components/apiKeys/ApiKeysArea.vue

```diff
@@ -290,6 +297,30 @@ export default class ApiKeysArea extends Vue {
         return HeaderState.DEFAULT;
     }
 
+    public get isHeaderShown(): boolean {
+        return !this.isEmpty || this.hasSearchQuery;
+    }
+
+    public get isDefaultHeaderState(): boolean {
+        return this.headerState === 0;
+    }
+
+    public get areApiKeysSelected(): boolean {
+        return !!(this.headerState === 1 && !this.isDeleteClicked);
```

Comment:
Why we are using double exclamation point here?
We are not comparing strings, so this seems redundant to me. Or there is another case?